src/test/Makefile

```diff
@@ -167,6 +167,7 @@ osd_unittest_SRC := \
 
 osd_unittest_DEFINES := \
 		USE_OSD \
+		USE_GPS \
```

Comment:
This needs to be `USE_GPS= \` to keep gcc from outputting `#define USE_GPS 1` for command line defines if no value is supplied. This will make the `#ifndef USE_GPS` below unnecessary.

Reply:
It's a shortcoming of how we are doing this really, all `USE_` defines in this `Makefile` should probably have a `=` at the end do avoid compiler errors when they are defined somewhere else.